php/class-customize-snapshot.php

```diff
@@ -228,8 +228,10 @@ public function post( $refresh = false ) {
 			'posts_per_page' => 1,
 			'post_type' => Customize_Snapshot_Manager::POST_TYPE,
 			'post_status' => array( 'draft', 'publish' ),
+			'no_found_rows' => true,
+			'ignore_sticky_posts' => true,
```

Comment:
Could also add:

```
'update_post_meta_cache' => false,
'update_post_term_cache' => false,
```

Because neither terms nor postmeta will be used, right?